### ISSUE TITLE
Tiny escape pods are made of ""nothing iron""

### ISSUE THREAD

**Issue by AdipemDragon** [tgstation/tgstation#81491]
Reporting client version: 514.1589

## Issue Summary
When deconstructed manually, the pod walls of small escape pods seem to yield iron that's "made out of nothing"

## Round ID:
[223989](https://scrubby.melonmesa.com/round/223989)

## Testmerges:
- [Temp keybinds patch for 515.1628](https://www.github.com/tgstation/tgstation/pull/81089)
- [A red spy has entered the base: Adds Spies, a roundstart antagonist inspired by Goonstation's Spy-Thief](https://www.github.com/tgstation/tgstation/pull/81231)

## Reproduction:
Deconstruct a small escape pod to make it into something nicer
Cry as the new iron doesn't stack with what you already had
![image](https://github.com/tgstation/tgstation/assets/76874615/c288cba2-e282-46b5-b91c-01ca386904a6)

**Comment by dhull2022** [comment]
Should be a consequence of #81109, right?

**Comment by AdipemDragon** [comment]
I mean, it's definitely linked i think, but they're the small version of the _roundstart_ escape pods, the issue at core here could be that these walls could've been weirdly varedited all this time and nobody knew
![image](https://github.com/tgstation/tgstation/assets/76874615/ae343802-8c49-4fb4-bd60-8ce2a5cedde0)

**Comment by AdipemDragon** [comment]
> Can you confirm that this is still happening?

I'll have to hang around rounds until it just happens to test again, give me anywhere between an hour and 4 days

**Comment by AdipemDragon** [comment]
Round 224285, the standard test:
![image](https://github.com/tgstation/tgstation/assets/76874615/7c31aa3b-5c19-4485-aab5-98156a5ce659)
Round 224287, the small pod:
![image](https://github.com/tgstation/tgstation/assets/76874615/5903e05e-85ef-4dfe-9b31-264d881f3f4f)
Round 224290, the big pod: 
![image](https://github.com/tgstation/tgstation/assets/76874615/003d14e2-b138-4b08-9b15-bdced3747984)
Round 224293, small pod on icebox:
![image](https://github.com/tgstation/tgstation/assets/76874615/ab6dbc59-a92f-41ce-aafd-a8c1f15bc405)

In conclusion: ????????? and also i give up
